### DISCUSSION TITLE
GitHub ssh key settings

### DISCUSSION THREAD

**Discussion by chanchoth** [community/community#40529] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there,

I have already configured my SSH in Ubuntu and connecting it to GitHub. When tested with:
ssh -T git@github.com
It said that successfully authenticated. When seeing SSH key in the Profile it is still showing the black color not green color. May you please help on how I can make this work or is it because I use 2FA? Thank you.

![ssh keys](https://user-images.githubusercontent.com/5266200/205014683-d5971558-f863-468d-9776-e5b7232f785c.png)

**Comment by airtower-luna** [comment]
Have you used the key for an actual Git operation (fetch, push, etc.)? I wouldn't be surprised if the authentication test doesn't update the "used when" status.

**Reply by chanchoth** [reply]
I use git push and pull. Is this correct way to do? Thank you.

**Reply by airtower-luna** [reply]
That's what I meant, yes. If the key is still marked as unused:

* Check if the repository you've been pushing/pulling actually uses an SSH URL. If it's an HTTPS URL, it can't use your SSH key. The easiest way to check is `git remote -v`.
* If you're sure it's an SSH URL, run Git with [`GIT_SSH_COMMAND="ssh -v"`](https://git-scm.com/docs/git#Documentation/git.txt-codeGITSSHCOMMANDcode) to see how SSH does the authentication.

**Reply by jricciardi** [reply]
Heya @chanchoth - did @airtower-luna's solution work for you? If it did, please let us know by marking this as the answer. Thank you!

**Comment by chanchoth** [comment]
Thank you so much for all your support. Please close this call.
